test/cypress/integration/Dashboard/component-add.spec.js

```diff
@@ -0,0 +1,54 @@
+function grabComponent(name) {
```

Comment:
If that is something that can be used in future tests somewhere else, it should be moved to the `index.js` file.

Reply:
It isn't now, so I can move it later 🤔 apart from that, here are all options tested and will be tested about adding a new component.

Reply:
Let's leave it here!